bigtable_rs/src/bigtable/read_rows.rs

```diff
@@ -33,6 +35,35 @@ pub async fn decode_read_rows_response(
     Ok(rows)
 }
 
+/// Decode the stream of `ReadRowsResponse` into a stream of `Result<(RowKey, Vec<RowCell>)>>`.
+pub async fn decode_read_rows_response_stream<'a>(
+    timeout: &'a Option<Duration>,
+    mut rrr: Streaming<ReadRowsResponse>,
+) -> Pin<Box<dyn Stream<Item = Result<(RowKey, Vec<RowCell>)>> + Send + 'a>> {
+    let started = Instant::now();
+
+    Box::pin(async_stream::stream! {
```

Comment:
Thanks, looks awesome. Do you mind putting a `use async_stream::stream;` above and use it here?